irispy/spectrograph.py

```diff
@@ -16,7 +16,27 @@
 
 
 class IRISSpectrograph(object):
-    """An object to hold data from multiple IRIS raster scans."""
+    """
+    A 2D IRIS Slit Jaw Imager Map.
```

Comment:
This is wrong.  IRISSPectrograph isn't a 2D Slit Jaw Map.  This should be replaces by L29.